gatsby-config.js

```diff
@@ -42,6 +42,23 @@ module.exports = {
           },
           keySeparator: '.',
           nsSeparator: '::',
+          react: {
```

Comment:
added basic support for parsing HTML elements beyond what is already included by default. Additional support is added in areas like `ContentBlock` where there are nested elements which i18next does not support out of the box.